### ISSUE TITLE
[Question] Gym has no attribute spaces

### ISSUE THREAD

**Issue by Shivam310** [openai/gym#3075]
### Question

Whenever I run my code I get this error:
```
AttributeError                            Traceback (most recent call last)
c:\Users\shiva\Documents\Atari_Breakout_RL_Project\.ipynb_checkpoints\Project_Atari_Breakout-checkpoint.ipynb Cell 2 in <cell line: 3>()
      [1](vscode-notebook-cell:/c%3A/Users/shiva/Documents/Atari_Breakout_RL_Project/.ipynb_checkpoints/Project_Atari_Breakout-checkpoint.ipynb#W1sZmlsZQ%3D%3D?line=0) #Test The Envrironment.
      [2](vscode-notebook-cell:/c%3A/Users/shiva/Documents/Atari_Breakout_RL_Project/.ipynb_checkpoints/Project_Atari_Breakout-checkpoint.ipynb#W1sZmlsZQ%3D%3D?line=1) import gym
----> [3](vscode-notebook-cell:/c%3A/Users/shiva/Documents/Atari_Breakout_RL_Project/.ipynb_checkpoints/Project_Atari_Breakout-checkpoint.ipynb#W1sZmlsZQ%3D%3D?line=2) from stable_baselines3 import A2C

File c:\Users\shiva\AppData\Local\Programs\Python\Python310\lib\site-packages\stable_baselines3\__init__.py:3, in <module>
      1 import os
----> 3 from stable_baselines3.a2c import A2C
      4 from stable_baselines3.common.utils import get_system_info
      5 from stable_baselines3.ddpg import DDPG

File c:\Users\shiva\AppData\Local\Programs\Python\Python310\lib\site-packages\stable_baselines3\a2c\__init__.py:1, in <module>
----> 1 from stable_baselines3.a2c.a2c import A2C
      2 from stable_baselines3.a2c.policies import CnnPolicy, MlpPolicy, MultiInputPolicy

File c:\Users\shiva\AppData\Local\Programs\Python\Python310\lib\site-packages\stable_baselines3\a2c\a2c.py:7, in <module>
      4 from gym import spaces
      5 from torch.nn import functional as F
----> 7 from stable_baselines3.common.on_policy_algorithm import OnPolicyAlgorithm
      8 from stable_baselines3.common.policies import ActorCriticCnnPolicy, ActorCriticPolicy, BasePolicy, MultiInputActorCriticPolicy
      9 from stable_baselines3.common.type_aliases import GymEnv, MaybeCallback, Schedule

File c:\Users\shiva\AppData\Local\Programs\Python\Python310\lib\site-packages\stable_baselines3\common\on_policy_algorithm.py:8, in <module>
...
---> 58 def __init__(self, num_envs: int, observation_space: gym.spaces.Space, action_space: gym.spaces.Space):
     59     self.num_envs = num_envs
     60     self.observation_space = observation_space

AttributeError: module 'gym' has no attribute 'spaces'
```
Here is my code:
```
import gym
from stable_baselines3 import A2C
from stable_baselines3.common.vec_env import VecFrameStack
from stable_baselines3.common.evaluation import evaluate_policy
from stable_baselines3.common.env_util import make_atari_env
from stable_baselines3.common.env_util import make_vec_env

import os
from gym.utils import play
from stable_baselines3.ddpg.policies import CnnPolicy

from ale_py import ALEInterface
from ale_py.roms import Breakout

ale = ALEInterface()


ale.loadROM(Breakout)

```

**Comment by pseudo-rnd-thoughts** [comment]
What version of gym are you using?

**Comment by pseudo-rnd-thoughts** [comment]
This looks like a stable-baselines-3 issue but Im confused why the error could occur. Also this is a super old version of gym so it is too late to update now

**Comment by Shivam310** [comment]
> This looks like a stable-baselines-3 issue but Im confused why the error could occur. Also this is a super old version of gym so it is too late to update now

Well I was confused too as restarting the machine fixed it also I had gym 0.25 installed but when I installed sb3 it uninstalled the 0.25 gym and installed 0.21 gym. I have now uninstalled sb3 and installed sb3 master. The issue no longer occurs.
